src/main/java/seedu/address/ui/RightSideBar.java

```diff
@@ -53,12 +56,15 @@ public void update(String commandText) {
     /**
      * Filters and returns the requiredPeriod according to the commandText
      */
-    private String requiredPeriod(String commandText) {
-        if (commandText.contains(PREFIX_PERIOD.toString())) {
+    private String requiredPeriod(CommandResult commandResult, String commandText) {
+        if (commandResult.equals(ViewSessionCommand.MESSAGE_SHOW_SESSIONS_SUCCESS)
```

Comment:
`commandResult.equals(ViewSessionCommand.MESSAGE_SHOW_SESSIONS_SUCCESS` will always return false because `commandResult` is a type of `CommandResult` and `ViewSessionCommand.MESSAGE_SHOW_SESSIONS_SUCCESS` is a type of String